ghost/admin/app/components/gh-members-segment-select.js

```diff
@@ -120,6 +120,26 @@ export default class GhMembersSegmentSelect extends Component {
             options.push(labelsGroup);
         }
 
+        const offers = yield this.store.query('offer', {limit: 'all'});
+
+        if (offers.length > 0 && !this.args.hideOffers) {
```

Comment:
Can the `if (!this.args.hideOffers)` conditional be used to avoid waiting on the API call when it's not needed?